lambdas/endpoint/package.json

```diff
@@ -12,14 +13,21 @@
     "@swc/jest": "^0.2.37",
     "@tsconfig/node20": "^20.1.4",
     "@types/jest": "^29.5.14",
-    "aws-lambda": "^1.0.7",
     "esbuild": "^0.24.0",
     "jest": "^29.7.0",
     "jest-mock-extended": "^3.0.7",
+    "tsx": "^4.19.2",
     "typescript": "^5.5.4"
   },
   "dependencies": {
+    "@aws-sdk/client-cognito-identity-provider": "^3.699.0",
     "@aws-sdk/client-dynamodb": "^3.696.0",
-    "@aws-sdk/lib-dynamodb": "^3.696.0"
+    "@aws-sdk/client-ses": "^3.699.0",
+    "@aws-sdk/lib-dynamodb": "^3.696.0",
+    "aws-lambda": "^1.0.7",
```

Comment:
The `aws-lambda` package can stay in dev dependencies if you import it using the `import type` syntax. Also - I think the intention is to be importing `@types/aws-lambda` rather than `aws-lambda` which is an annoyingly named 3rd party CLI tool. this was my fault as I installed it originally in #183